src/bundle/Resources/public/js/scripts/helpers/modal.helper.js

```diff
@@ -50,4 +50,54 @@ const controlManyZIndexes = (items, listenerContainer) => {
     };
 };
 
-export { controlZIndex, controlManyZIndexes };
+const showModalLoader = ({ headerText, descriptionText, modalNode }) => {
```

Comment:
personally I would put `modalNode` as its own method argument, as it's THE element which is mandatory for it to even work?
Same with `hideModalLoader` (there it would be even more useful, as there would be no need to pass object just to pass one argument